docs/autogen.py

```diff
@@ -418,17 +425,24 @@ def post_process_signature(signature):
     return signature
 
 
+def clean_module_name(name):
+    if name.startswith('keras_applications'):
```

Comment:
have you considered using regex here to replace the '_' to '.'

Reply:
comes with keras-team/keras. Avoided to change.